#get wifi mode
---
int8 mode